core/names.js

```diff
@@ -72,9 +115,18 @@ Blockly.Names.prototype.reset = function() {
  * @return {string} An entity name legal for the exported language.
```

Comment:
"A legal entity name for the exported language." or "An entity name, legal in the exported language."

(unrelated, but...  :/  )

Reply:
I think it's technically correct english!  But yeah, I can change that.

Reply:
Agreed, but my brain backfires when trying to read the original form.